### DISCUSSION TITLE
Voxelab Aquila (Ender V2 clone) is using Jyes

### DISCUSSION THREAD

**Discussion by joeclifford** [Jyers/Marlin#822] [General Discussion]
I noticed in CHEPs video on the Voxelab Aquila (Ender V2 clone), that they are using Jyers for their firmware - https://github.com/alexqzd/Marlin/releases

They are using the screen in landscape mode. I 'm wondering if we can use this as a basis for lanscape mode in Jyers.

**Comment by tiagofreire-pt** [comment]
https://github.com/Jyers/Marlin/discussions/751

https://github.com/Jyers/Marlin/discussions/653

**Comment by Jyers** [comment]
@joeclifford I don't believe that fork is in landscape mode, they just added config versions for the Aquila expecting people to change the mounting I suppose. That being said, landscape mode is in the works.

**Reply by coderRPN** [reply]
Hi @Jyers just dropping in to check if we have landscape mode available yet?

**Reply by coderRPN** [reply]
@tititopher68-dev wow looks great. I can compile it myself is there any specific steps to follow? Thanks in advance
